android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -76,6 +76,7 @@ protected void publishResults(CharSequence constraint, FilterResults results) {
     public SpeakersListAdapter(List<Speaker> speakers, Activity activity) {
         super(speakers);
         this.activity = activity;
+        this.picassoCache = new PicassoCache(activity.getApplicationContext());
```

Comment:
I am using application context for building picasso object. So it is needed. For avoiding memory leaks i will make weak reference for context.

From this [blog](https://android-developers.googleblog.com/2009/01/avoiding-memory-leaks.html) application context is more safer than activity context :+1: